src/js/node/assert.ts

```diff
@@ -1,1271 +1,998 @@
-        assert === void 0 && (assert = require_assert()), assert(typeof name == "string", "'name' must be a string");
-        var determiner;
-        typeof expected == "string" && startsWith(expected, "not ")
-          ? ((determiner = "must not be"), (expected = expected.replace(/^not /, "")))
-          : (determiner = "must be");
-        var msg;
-        if (endsWith(name, " argument"))
-          msg = "The ".concat(name, " ").concat(determiner, " ").concat(oneOf(expected, "type"));
-        else {
-          var type = includes(name, ".") ? "property" : "argument";
-          msg = 'The "'.concat(name, '" ').concat(type, " ").concat(determiner, " ").concat(oneOf(expected, "type"));
-        }
-        return (msg += ". Received type ".concat(_typeof(actual))), msg;
-      },
-      TypeError,
-    );
-    createErrorType(
-      "ERR_INVALID_ARG_VALUE",
-      function (name, value) {
-        var reason = arguments.length > 2 && arguments[2] !== void 0 ? arguments[2] : "is invalid";
-        var inspected = util.inspect(value);
-        return (
-          inspected.length > 128 && (inspected = "".concat(inspected.slice(0, 128), "...")),
-          "The argument '".concat(name, "' ").concat(reason, ". Received ").concat(inspected)
-        );
-      },
-      TypeError,
-      RangeError,
-    );
-    createErrorType(
-      "ERR_INVALID_RETURN_VALUE",
-      function (input, name, value) {
-        var type;
-        return (
-          value && value.constructor && value.constructor.name
-            ? (type = "instance of ".concat(value.constructor.name))
-            : (type = "type ".concat(_typeof(value))),
-          "Expected ".concat(input, ' to be returned from the "').concat(name, '"') +
-            " function but got ".concat(type, ".")
-        );
-      },
-      TypeError,
-    );
-    createErrorType(
-      "ERR_MISSING_ARGS",
-      function () {
-        for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; _key++)
-          args[_key] = arguments[_key];
-        assert === void 0 && (assert = require_assert()),
-          assert(args.length > 0, "At least one arg needs to be specified");
-        var msg = "The ",
-          len = args.length;
-        switch (
-          ((args = args.map(function (a) {
-            return '"'.concat(a, '"');
-          })),
-          len)
-        ) {
-          case 1:
-            msg += "".concat(args[0], " argument");
-            break;
-          case 2:
-            msg += "".concat(args[0], " and ").concat(args[1], " arguments");
-            break;
-          default:
-            (msg += args.slice(0, len - 1).join(", ")), (msg += ", and ".concat(args[len - 1], " arguments"));
-            break;
-        }
-        return "".concat(msg, " must be specified");
-      },
-      TypeError,
-    );
-    module2.exports.codes = codes;
-  },
-});
+    if (argsLen === 2) operator = "!=";
+  }
+
+  if (message instanceof Error) throw message;
```

Comment:
instanceof can throw

can we do `Error.isError(message)` instead

Reply:
Done